tests/foreman/api/test_permission.py

```diff
@@ -272,13 +303,21 @@ def test_positive_check_read(self):
 
         :CaseImportance: Critical
         """
-        for entity_cls in (entities.Architecture, entities.Domain):
+        for entity_cls in (
+                entities.Architecture,
+                entities.Domain,
+                entities.ActivationKey):
             with self.subTest(entity_cls):
-                entity_id = entity_cls().create().id
+                entity = entity_cls()
+                if ('organization' in entity.get_fields()
+                        and 'location' in entity.get_fields()):
+                    entity.organization = [self.org]
```

Comment:
So for this and all the later tests `OneToManyField` checks aren't needed anymore? I don't get it, the list of entities is the same, why different approach?